src/main/java/beans/exceptions/BaseExceptionResponseBean.java

```diff
@@ -10,6 +10,8 @@
     protected String status;
     protected String url;
     protected String type;
+    // For backwards compatibility with old CloudCare, we need to add the message property
```

Comment:
unfortunately (or fortunately) both old and new cloudcare utilize the same form entry javascript code and that javascript code expects `message`. we could just change the JS to look for `.exception` as well